pinot-query-runtime/src/test/java/org/apache/pinot/query/QueryTestSet.java

```diff
@@ -26,6 +26,11 @@ public class QueryTestSet {
   @DataProvider(name = "testSql")
   public Object[][] provideTestSql() {
     return new Object[][]{
+        // Failed left join. Not sure what's the expectation here.
+        // new Object[]{"SELECT a.col1 FROM a LEFT JOIN b ON (a.col1 = 'foo')"},
+        // new Object[]{"SELECT a.col1 FROM a LEFT JOIN b ON (a.col1 = b.col1 and a.col1 = 'foo)"},
+        // Failed semi join. Somehow it gets time out now
+        // new Object[]{"SELECT * FROM b WHERE b.col1 IN (SELECT a.col1 FROM a)"},
```

Comment:
let's dont add these to the E2E test. but to the operator test

Reply:
Removed